ui/app/components/auth-config-form/options.js

```diff
@@ -65,4 +66,8 @@ export default class AuthConfigOptions extends AuthConfigComponent {
     this.router.transitionTo('vault.cluster.access.methods').followRedirects();
     this.flashMessages.success('The configuration was saved successfully.');
   }
+
+  get getLoginLink() {
+    return `${window.origin}/ui/vault/auth?with=${encodePath(this.args.model.path)}`;
```

Comment:
good find! But since slashes could be part of the mount name we I think we want those encoded here, so this can just use `encodeURIComponent` directly